backend/src/main/java/com/hyooun/wilson/review/domain/Reviewer.java

```diff
@@ -0,0 +1,17 @@
+package com.hyooun.wilson.review.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
```

Comment:
Reviewer 클래스에 Getter annotation이 누락되어 있습니다. Lombok의 @Getter를 추가해주세요.

```suggestion
@Getter 추가
```